cmd/config/internal/commands/cmdlistsetters.go

```diff
@@ -107,7 +108,7 @@ func (r *ListSettersRunner) ListSetters(w io.Writer, openAPIPath, resourcePath s
 		return err
 	}
 	table := newTable(w, r.Markdown)
-	table.SetHeader([]string{"NAME", "VALUE", "SET BY", "DESCRIPTION", "COUNT", "REQUIRED"})
+	table.SetHeader([]string{"NAME", "VALUE", "IS SET", "SET BY", "DESCRIPTION", "COUNT", "REQUIRED"})
```

Comment:
I think this should be next to `REQUIRED` column. `REQUIRED` indicates the user that the setter is required. `IS SET` indicates that it is set.

Reply:
Done.